src/pages/Main.js

```diff
@@ -44,8 +44,8 @@ export default function MainPage() {
 
   return (
     <>
-      <main>
-        <header>
+      <main className="main-main">
+        <header className="main-header
```

Comment:
```suggestion
      <main className="main-main">
        <header className="main-header">
```
클래스명 수정하면서 >가 빠진거같아요 !